signac/contrib/job.py

```diff
@@ -214,42 +210,30 @@ def statepoint(self):
             `sp_dict = job.statepoint()` instead of `sp = job.statepoint`.
             For more information, see :class:`~signac.JSONDict`.
         """
-        if self._sp is None:
-            self._sp = SyncedAttrDict(self._statepoint, parent=_sp_save_hook(self))
-        return self._sp
+        return self._statepoint
 
     @statepoint.setter
     def statepoint(self, new_sp):
         self._reset_sp(new_sp)
 
-    @property
-    def sp(self):
-        """ Alias for :attr:`Job.statepoint`.
-
-        .. warning::
-
-            As with :attr:`Job.statepoint`, use `job.sp()` instead of
-            `job.sp` if you need a deep copy that will not modify the
-            underlying persistent JSON file.
-        """
-        return self.statepoint
-
-    @sp.setter
-    def sp(self, new_sp):
-        self.statepoint = new_sp
+    sp = statepoint  # alias
```

Comment:
Will this still appear in the documentation? Why not do the same thing to `doc` / `document`?

Reply:
Yes, but I've reverted this, so that we can give it a distinct doc-string.